website/docs/r/instance.html.markdown

```diff
@@ -224,6 +224,7 @@ This resource supports the following arguments:
 * `enable_primary_ipv6` - (Optional) Whether to assign a primary IPv6 Global Unicast Address (GUA) to the instance when launched in a dual-stack or IPv6-only subnet. A primary IPv6 address ensures a consistent IPv6 address for the instance and is automatically assigned by AWS to the ENI. Once enabled, the first IPv6 GUA becomes the primary IPv6 address and cannot be disabled. The primary IPv6 address remains until the instance is terminated or the ENI is detached. Disabling `enable_primary_ipv6` after it has been enabled forces recreation of the instance.
 * `enclave_options` - (Optional) Enable Nitro Enclaves on launched instances. See [Enclave Options](#enclave-options) below for more details.
 * `ephemeral_block_device` - (Optional) One or more configuration blocks to customize Ephemeral (also known as "Instance Store") volumes on the instance. See [Block Devices](#ebs-ephemeral-and-root-block-devices) below for details. When accessing this as an attribute reference, it is a set of objects.
+* `force_destroy` - (Optional) Destroys instance even if `disable_api_termination` or `disable_api_stop` is set to `true`. Defaults to `false`.
```

Comment:
There's some wording in the S3 Bucket documentation of its [`force_destroy` argument](https://github.com/hashicorp/terraform-provider-aws/blob/5e307d3e368005dd844ccd36fc53a041a17944b0/website/docs/r/s3_bucket.html.markdown?plain=1#L39) about having to run `terraform apply` before `terraform destroy`. Do we want to add something similar here?

Reply:
@ewbankkit added similar doc to `s3 Bucket`